packages/components/src/components/atoms/BaseCard/BaseCard.tsx

```diff
@@ -95,8 +95,8 @@ export const BaseCard: FunctionComponent<BaseCardProps> = ({
   tableBoundaries,
   disableNativeEvents,
   position = { x: 0, y: 0 },
-  onPositionChanged = () => {},
-  onFlipped = () => {},
+  onPositionChanged,
+  onFlipped,
```

Comment:
Why did you have to change this? If you need to change it then maybe we should also set the property as not optional 😄

Reply:
Yeah, with the default value for callbacks, Storybook actions (the panel that logs on event activation) seems to not work.

The events should still to be optional IMO. Why exactly do you want to make them not optional?